SpatialGDK/Extras/fastbuild/install.ps1

```diff
@@ -6,7 +6,7 @@ Import-Module BitsTransfer
 Add-Type -AssemblyName System.IO.Compression.FileSystem
 
 $fileshare="\\lonv-file-01"
-$version="v0.96"
+$version="v0.98"
```

Comment:
What about the uninstall script? I'd copy the current 0.96 version and name it `uninstall-0.96.ps1` so people can both move from 0.96, and also uninstall/reinstall 0.98

Reply:
Good idea, done.